var/spack/repos/builtin/packages/simulationio/package.py

```diff
@@ -23,6 +23,8 @@ class Simulationio(CMakePackage):
     variant('pic', default=True,
```

Comment:
Can you add descriptions to these variants? And rename develop to master?

Reply:
Done.